core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -583,7 +583,7 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
   /**
    * Runs this action in a new fiber, resuming when the fiber terminates.
    *
-   * If the fiber fails with an error it will be captured in Right side of the error Either
+   * If the fiber fails with a failure value it will be captured in Right side of the error Either
```

Comment:
Here we could rephrase as such:
```
If the fiber fails, the cause will be captured in the error channel.
```
anticipating that the type of `sandbox` is going to be `IO[Cause[E], A]` very soon.

cc @jdegoes

Reply:
👍